biblio/how_to_cite_gudhi.bib.in

```diff
@@ -2,141 +2,157 @@ @book{gudhi:urm
 , title        = "{GUDHI} User and Reference Manual"
 , author      = "{The GUDHI Project}"
 , publisher     = "{GUDHI Editorial Board}"
-, year         = 2015
-, url =    "http://gudhi.gforge.inria.fr/doc/latest/"
+, edition =     "{@GUDHI_VERSION@}"
+, year         = 2019
```

Comment:
Can't we also generate the year, while we are there?

Reply:
Year is now also generated 518de123bcf19a85accca5660844355da1106511